search-ui/src/components/search/SearchResults.vue

```diff
@@ -1,15 +1,16 @@
-<template>
+<template>   
   <base-table
     v-if="search.searchType === 'business'"
-    class="mt-10"
+    class="mt-4"
```

Comment:
You'll want this on the owner table too OR actually probably better to wrap the tables in a div and add this class there